File: ApiSuvesaPos/Negocio/Modelo/cxc/abonoccobrar/abonoccobrar.vb
Namespace Modelo
    Public Class abonoccobrar
        Public Property Id_Recibo As Long
        Public Property Num_Recibo As Long
        Public Property Cod_Cliente As Long
        Public Property Nombre_Cliente As String
        Public Property Saldo_Cuenta As Decimal
        Public Property Monto As Decimal
        Public Property Saldo_Actual As Decimal
        Public Property Fecha As DateTime
        Public Property Observaciones As String
        Public Property Anula As Boolean
        Public Property Ced_Usuario As String
        Public Property Contabilizado As Boolean
        Public Property Asiento As Long
        Public Property Cod_Moneda As Integer
        Public Property Cheque As String
        Public Property Num_Caja As Integer
        Public Property banco As String
        Public Property Num_Apertura As Integer
        Public Property EstadoEnvio As String
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/cxc/abonoccobrar/abonoccobrar.cs
using System;

namespace Modelo
{
    public class abonoccobrar
    {
        public long Id_Recibo { get; set; }
        public long Num_Recibo { get; set; }
        public long Cod_Cliente { get; set; }
        public string Nombre_Cliente { get; set; }
        public decimal Saldo_Cuenta { get; set; }
        public decimal Monto { get; set; }
        public decimal Saldo_Actual { get; set; }
        public DateTime Fecha { get; set; }
        public string Observaciones { get; set; }
        public bool Anula { get; set; }
        public string Ced_Usuario { get; set; }
        public bool Contabilizado { get; set; }
        public long Asiento { get; set; }
        public int Cod_Moneda { get; set; }
        public string Cheque { get; set; }
        public int Num_Caja { get; set; }
        public string banco { get; set; }
        public int Num_Apertura { get; set; }
        public string EstadoEnvio { get; set; }
    }
}
